.github/workflows/license_finder.yml

```diff
@@ -36,7 +36,9 @@ jobs:
       - name: Generate requirements.txt (exclude dev dependencies)
         run: |
           poetry export -f requirements.txt --without-hashes > requirements.txt
+          pip install -r requirements.txt
 
       - name: Run license finder
         run: |
-          poetry run license_finder --python-version=3
+          /opt/hostedtoolcache/Ruby/*/x64/lib/ruby/gems/*/gems/license_finder-*/bin/license_finder_pip.py requirements.txt
+          poetry run license_finder
```

Comment:
Do we need `poetry run` here? It seems to pass just fine if we simply call `license_finder`. For some reason it fails if we use `--python-version=3`